.changeset/short-cooks-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+'lit-html': patch
```

Comment:
Nit: Please add lit monorepo patch

I.e.:

```
'lit-html': patch
'lit': patch
```

Reply:
THANK YOU! 😌